common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -59,29 +59,68 @@ switches.  Like with the earlier fences, another fence is erected 20m
 out which will again switch the vehicle to LAND (HOLD for Rover), if it continues away from
 home.
 
+Plane Actions
+-------------
+
+If, enabled, and  the fence radius (:ref:`FENCE_RADIUS<FENCE_RADIUS>`) or altitudes,if non-zero, (:ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` and :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>`) are exceeded, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` below:
+
++------------+--------------------+
```

Comment:
This table is not accurate for current master.

For reference: https://github.com/ArduPilot/ardupilot/blob/master/libraries/AC_Fence/AC_Fence.cpp#L40

Reply:
copy paste error...thanks for catching